apps/js_urls/serializer.py

```diff
@@ -0,0 +1,130 @@
+# Python Standard Library Imports
+import json
+from urllib.parse import urljoin
+
+# Django Imports
+from django.urls import get_resolver
+from django.urls.resolvers import (
+    RegexPattern,
+    RoutePattern,
+    URLPattern,
+    URLResolver,
+)
+
+# HTK Imports
+from htk.apps.js_urls.constants.regexp import (
+    URL_ARG_RE,
+    URL_KWARG_RE,
+    URL_OPTIONAL_CHAR_RE,
+    URL_OPTIONAL_GROUP_RE,
+    URL_PATH_RE,
+)
+from htk.apps.js_urls.utils import replace
+from htk.utils import htk_setting
+
+
+def get_urls_as_dict(resolver=None):
+    resolver = resolver or get_resolver()
+    return dict(_parse_resolver(resolver))
+
+
+def get_urls_as_json(resolver=None):
+    resolver = resolver or get_resolver()
+    return json.dumps(get_urls_as_dict(resolver))
+
+
+def _parse_resolver(
+    resolver, current_namespace=None, url_prefix=None, include_all=False
+):
+    allowed_urls = htk_setting('HTK_APPS_JS_URLS_ALLOWED')
+    namespace = (
+        f"{current_namespace}:{resolver.namespace}"
+        if current_namespace
+        else resolver.namespace
+    )
```

Comment:
For this pattern, prefer multiple lines instead of a ternary, splitting out the `prefix`. If we also needed to have a suffix, then you could add the `suffix` variable.

```
prefix = f'{current_namespace}:' if current_namespace else ''
namespace = f'{prefix}{resolver.namespace}'
```

Reply:
I tried this and it is not working because if there is no namespace, `resolver.namespace` is `None` and with using `f` string, it adds as `None` instead of an empty string and becomes something in the screenshot below:

![image](https://github.com/hacktoolkit/django-htk/assets/1487670/5b036a7d-177e-42bf-9147-462ce2c6c544)